client/hwlib/debian/vendor-rust.sh

```diff
@@ -11,6 +11,9 @@ if [ ! $(which cargo-vendor-filterer) ]; then
     exit 3
 fi
 
+[ -e /usr/bin/jq ] || (echo "jq is required to run this script. Try installing it with 'sudo apt install jq'" && exit 1);
```

Comment:
Instead of assuming the path, use `test -n "$(command -v jq)"`

In my Makefiles I often use the form:
```make
JQ ?= $(or $(shell command -v jq),$(error jq is required to blargh))

foo:
    $(JQ) ...
```

This way I can both check and control the executable externally with ease.